meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh

```diff
@@ -305,6 +317,10 @@ board_config() {
               DASHARO_REL_VER_DES="1.1.3"
               BIOS_LINK_COMM="${FW_STORE_URL}/${DASHARO_REL_NAME}/v${DASHARO_REL_VER}/${DASHARO_REL_NAME}_v${DASHARO_REL_VER}_ddr4.rom"
               BIOS_LINK_DES="${FW_STORE_URL_DES}/MS-7D25/v${DASHARO_REL_VER_DES}/${DASHARO_REL_NAME}_v${DASHARO_REL_VER_DES}_ddr4.rom"
+              HAVE_HEADS_FW="true"
```

Comment:
@miczyg1 for next v1.2.19 DTS release we only want to provide heads support on NV4X ADL, so this heads related variables should be commented out for MSI targets right?

Reply:
@miczyg1 ok, thanks. Done https://github.com/Dasharo/meta-dts/pull/99/commits/ea4834fff6d02156c43c902ef38f9189304ed677